### ISSUE TITLE
Dice not working in Chitubox

### ISSUE THREAD

**Issue by aaronweatherall** [accassid/dice-gen#132]
Hey mate, (Patreon subscriber here!) I'm having issues with Premium Chitubox (and Fusion 360) opening dice made in the app!

<img width="275" alt="image" src="https://user-images.githubusercontent.com/980325/219933799-83f2c449-13eb-4cae-86b0-6a847f4c3531.png">

<img width="308" alt="image" src="https://user-images.githubusercontent.com/980325/219933880-228b3d07-d75d-4805-bb34-4a67d8bc0655.png">

Basically the models don't seem to be closed for whatever reason. I've tried numerous fonts and opening them in Fusion360 and even there it says the models aren't closed.

Are there any issues you can see from your end?

The font I'm testing with is Gugi (in your list of Google fonts).
